README.md

```diff
@@ -1,4 +1,38 @@
 # NeuroBridge1.0
-neurobridges.org
+Deployed site: neurobridges.org
 
 Rebuilt with React / GatsbyJS / Material UI
+## Description
+
+This website is intended to give information and access to the Neurobridge Portal Application. The site was built using GatsbyJS and Material UI and is deployed on Netlify. The site can be found at neurobridges.org
+
+
+## Installation
+
+- To run this website locally, clone the repo and install the dependencies (`npm i`)
+- The content for this website is hosted inside this repository under the content folder in yaml format.
+- To spin up the application, run  `npm start` which runs gatsby develop and will allow you to access the site on localhost:8000
+- You will also be able to access the data coming through from contentful by checking out localhost:8000/_graphql
+- To check the build, run `npm build` which will run gatsby build and build the site. To see the site, run `gatsby serve`, and you should be able to see the site on localhost:9000
```

Comment:
This is a good bit of info to provide. i think we should stress the point of checking the build. Maybe something like:

> When proposing changes, always verify the build process runs without error locally with `npm run build`. Serve the built application on port 9000 with `gatsby serve`.